app/models/company.rb

```diff
@@ -2,7 +2,7 @@ class Company < ApplicationRecord
 
   include HasSwedishOrganization
 
-  validates_presence_of :name, :company_number
+  validates_presence_of :company_number
   validates_presence_of :company_number
```

Comment:
A user is not required to provide a company name when they fill out their application.  So when an admin approves it and we have to instantiate a company, we don't have a name.  
IMHO, the right thing to do is to require the user to provide us with a company name when they fill out the application.  (but after MVP)